pkg/labels/selector.go

```diff
@@ -809,3 +809,77 @@ func SelectorFromSet(ls Set) Selector {
 func ParseToRequirements(selector string) ([]Requirement, error) {
 	return parse(selector)
 }
+
+// Conflicts takes 2 maps
```

Comment:
When you view the godoc as rendered html in a web browser, it shows up like this:

"Conflicts takes 2 maps returns true if there a key match between the maps but the value doesn't match returns false in other cases"

It runs together. Please update accordingly.